modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - New `expand` function added to `tinymce.selection` which expands the selection around the nearest word. #TINY-9001
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
+- New `color_foreground_default` and `color_background_default` options to set the initial default color for the `forecolor` and `backcolor` toolbar buttons. #TINY-9183
```

Comment:
This needs updating to match the new names. We also should mention it applies to both menu items and toolbar buttons.
```suggestion
- New `color_default_foreground` and `color_default_background` options to set the initial default color for the `forecolor` and `backcolor` toolbar buttons and menu items. #TINY-9183
```